SETUP.md

```diff
@@ -186,6 +186,18 @@ This is configured via the [`.ci-mgmt.yaml`](.ci-mgmt.yaml) file and can be re-g
 
 The default configuration is designed for third-party use and will need to be adjusted for Pulumi internal providers.
 
+### CI Management
+
+When you have set up your provider with this template, your `.github/workflows` folder will contain a `resync-build.yml` Workflow
+which runs a once-a week cronjob to upgrade the workflow files to match the latest process for Pulumi providers.
+
+The Workflow will open a pull request so you can review the changes, make adjustments to your `.ci-mgmt.yaml` config file, and merge.
+
+We recommend setting `PULUMI_PROVIDER_AUTOMATION_TOKEN` to a token with Write scope permissions on your repository so that acceptance tests run on these automated pull requests.
```

Comment:
@guineveresaenger to avoid confusion, I would list exactly what is needed. If people only would set `contents: write` permission on the repo, that would not be enough. It also requires `pull-requests: write` permission.

https://github.com/pulumi/pulumi-tf-provider-boilerplate/blob/main/.github/workflows/resync-build.yml#L10-L11

Below is an image of how my token for Pulumiverse providers is setup for my fine grained Github access token.

<img width="412" alt="Screenshot 2025-02-26 at 08 17 13" src="https://github.com/user-attachments/assets/b7e8113b-a431-405b-bc15-3cadf45764f7" />